platform/ios/Makefile

```diff
@@ -4,6 +4,10 @@ project_path = PolyPodApp/PolyPod.xcodeproj
 scheme_name = PolyPod
 export_options_plist = exportOptions.plist
 destination = 15.2
+appstore_api_key_id = $(POLYPOD_APPLE_IOS_APPSTORE_API_KEY_ID)
```

Comment:
Nit: Any reason to have  local variable for `POLYPOD_APPLE_IOS_APPSTORE_API_KEY_ID` but not for other values like `POLYPOD_APPLE_IOS_APPSTORE_P12_PASS`?

Reply:
Yeah, we're using them as actual environment variables, if I specify something here I can only use it as a make variable - unless we do the whole `$(value)` game of course. I probably would have done that if those variables would be used more than once, but given how they're only used a single time, seemed cleaner to just use them where they're needed, once. These here are used a couple of times so I left them.